### ISSUE TITLE
[Feature Proposal] Configurable background positions

### ISSUE THREAD

**Issue by benface** [tailwindcss/tailwindcss#608]
There is already a `backgroundSize` key in the config object, I suggest adding `backgroundPosition` as well to customize the generated background position utilities. The default value would be:

```
{
    'bottom': 'bottom',
    'center': 'center',
    'left': 'left',
    'left-bottom': 'left bottom',
    'left-top': 'left top',
    'right': 'right',
    'right-bottom': 'right bottom',
    'right-top': 'right top',
    'top': 'top',
}
```

**Comment by adamwathan** [comment]
I'm on board with this 👍🏻 Happy to check out a PR or I can try and get it in in a few weeks.

**Comment by benface** [comment]
Thanks @adamwathan, done: #609

**Comment by adamwathan** [comment]
Merged, will release in 0.8.0 today or tomorrow 👍
